data/en/census_household.json

```diff
@@ -421,12 +421,12 @@
                             "hide_guidance": "Hide further guidance",
                             "content": [{
                                 "list": [
-                                    "Select the relationships between members of your household",
-                                    "If you are the parents of adopted children, select the ‘Mother or father’ option to show your relationship to them.",
-                                    "If you have foster children living with you, select the ‘Unrelated’ option.",
-                                    "Include half-brothers and half-sisters in the ‘Step-brother or step-sister’ category.",
-                                    "Same-sex civil partner is used to describe the relationship between two people who have legally registered a civil partnership.",
-                                    "If none of the options fully reflects your relationship, please select the one which you feel best describes your situation."
+                                    "select the relationships between members of your household",
+                                    "if you are the parents of adopted children, select the ‘Mother or father’ option to show your relationship to them.",
+                                    "if you have foster children living with you, select the ‘Unrelated’ option.",
+                                    "include half-brothers and half-sisters in the ‘Step-brother or step-sister’ category.",
+                                    "same-sex civil partner is used to describe the relationship between two people who have legally registered a civil partnership.",
+                                    "if none of the options fully reflects your relationship, please select the one which you feel best describes your situation."
```

Comment:
these shouldnt be lower case as not an include list but to be honest it needs re-writing but could we move to guidance under question not in panel please